sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -809,6 +827,59 @@ private case class GpuParquetFileFilterHandler(@transient sqlConf: SQLConf) {
     }
   }
 
+  // scalastyle:off
+  // Implement Parquet LIST backwards-compatibility rules.
+  // See: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#backward-compatibility-rules
+  // scalastyle:on
+  // Inspired by Apache Spark's ParquetSchemaConverter.isElementType().
```

Comment:
add a permalink to the "inspiring" code

Reply:
Agreed. I've added a link to the inspiration code.